### PR TITLE
adding delete action to elbv2

### PR DESCRIPTION
**Pull request by joshuaroot**
- update changelog
- adding delete action to elbv2 module

Signed-off-by: Joshua Root <joshua.root@capitalone.com>

### CODE REVIEW

**Comment by Lawouach** [comment]
It seems by merging the previous PR, we have broken this one. Could you please rebase and push again?

**Comment by joshuaroot** [comment]
looks like it was just out of sync. no need to rebase as a quick branch update worked.

File: chaosaws/elbv2/actions.py [hunk 1]
```diff
@@ -126,6 +126,27 @@ def set_subnets(load_balancer_names: List[str],
     return results
 
 
+def delete(load_balancer_names: List[str],
```
Comment (Lawouach) [inline chaosaws/elbv2/actions.py]: Would it be okay to be a little more explicit in the name of the action?

File: chaosaws/elbv2/actions.py [hunk 2]
```diff
@@ -126,6 +126,27 @@ def set_subnets(load_balancer_names: List[str],
...
+    client = aws_client('elbv2', configuration, secrets)
+    load_balancers = get_load_balancer_arns(load_balancer_names, client)
+
+    for k, v in load_balancers.items():
+        if k not in ('application', 'network'):
+            continue
+
+        for l in v:
+            logger.debug('Deleting load balancer %s' % l)
+            client.delete_load_balancer(LoadBalancerArn=l)
```
Comment (Lawouach) [inline chaosaws/elbv2/actions.py]: Does this return anything we should at least log? Could be handy when things go wrong...

**Comment by Lawouach** [comment]
We tend to favour rebase over merge branches for next time :)

File: chaosaws/elbv2/actions.py (hunk 2 above)
Comment (joshuaroot) [reply chaosaws/elbv2/actions.py]: from the doc, there is no return on a successful delete, just the `ResponseMetadata`

Comment (Lawouach) [reply chaosaws/elbv2/actions.py]: I see. Well, it is what it is :)

Thanks.

**Pull request merged**